Fasten JMAP Draft memory integration testing

chibenwa opened the pull request:
Before:

![Capture d’écran de 2020-04-01 09-30-13](https://user-images.githubusercontent.com/6928740/78093815-3cab4980-73fd-11ea-83fa-728b05486389.png)

After:

![Capture d’écran de 2020-04-01 09-31-12](https://user-images.githubusercontent.com/6928740/78093828-42a12a80-73fd-11ea-8dc8-6d4846ba8b88.png)

mbaechler commented on server/container/guice/protocols/jmap-draft/src/test/java/org/apache/james/modules/TestJMAPServerModule.java:
```diff
@@ -107,6 +109,13 @@ private TestJMAPServerModule(long maximumLimit) {
     protected void configure() {
         bindConstant().annotatedWith(Names.named(GetMessageListMethod.MAXIMUM_LIMIT)).to(maximumLimit);
+
+        bind(RetryBackoffConfiguration.class)
```
is it related to JMAPServer only?

chibenwa replied:
No not really

chibenwa commented:
`CassandraAccessTokenRepositoryTest>AccessTokenRepositoryTest.validTokenMustBeRetrieved:45 » InvalidAccessToken`

chibenwa commented:
```
[INFO] Apache James Project ............................... SUCCESS [  2.510 s]
[INFO] Apache James :: Testing :: Base .................... SUCCESS [  1.982 s]
[INFO] Apache James :: Server ............................. SUCCESS [  0.249 s]
[INFO] Apache James :: Server :: Common Utilities ......... SUCCESS [ 14.271 s]
[INFO] Apache James :: Core Domain Objects ................ SUCCESS [  5.396 s]
[INFO] Apache James :: Server :: Task :: API .............. SUCCESS [  8.655 s]
[INFO] Apache James :: Json serialization ................. SUCCESS [  3.594 s]
[INFO] Apache James :: Server :: Task :: Json ............. SUCCESS [  3.480 s]
[INFO] Apache James :: Event Sourcing ..................... SUCCESS [  0.114 s]
[INFO] Apache James :: Event sourcing :: pojo ............. SUCCESS [  8.975 s]
[INFO] Apache James :: Event Sourcing :: Event Store :: API SUCCESS [  8.925 s]
[INFO] Apache James :: Event sourcing :: core ............. SUCCESS [  9.932 s]
[INFO] Apache James :: Event sourcing :: Event Store :: Memory SUCCESS [  8.195 s]
[INFO] Apache James :: Server :: Lifecycle API ............ SUCCESS [  2.660 s]
[INFO] Apache James :: Server :: Task :: Memory implementation SUCCESS [ 51.847 s]
[INFO] Apache JAMES backends common ....................... SUCCESS [  0.112 s]
[INFO] Apache James Cassandra backend ..................... SUCCESS [02:19 min]
[INFO] Apache James :: Metrics ............................ SUCCESS [  0.126 s]
[INFO] Apache James :: Metrics :: API ..................... SUCCESS [  1.093 s]
[INFO] Apache James :: Metrics :: Tests ................... SUCCESS [  9.433 s]
[INFO] Apache James :: Mailbox ............................ SUCCESS [  0.073 s]
[INFO] Apache James :: Mailbox :: API ..................... SUCCESS [ 13.692 s]
[INFO] Apache James :: Mailets parent and aggregator ...... SUCCESS [  0.093 s]
[INFO] Apache James :: Mailet API ......................... SUCCESS [  4.311 s]
[INFO] Apache James :: Test helpers for Mailet ............ SUCCESS [  4.912 s]
[INFO] Apache James :: Server :: Testing .................. SUCCESS [  3.983 s]
[INFO] apache-james-backends-es ........................... SUCCESS [02:38 min]
[INFO] apache-james-backends-jpa .......................... SUCCESS [  1.266 s]
[INFO] Apache James RabbitMQ backend ...................... SUCCESS [04:45 min]
[INFO] Apache James :: Event sourcing :: Event Store :: Cassandra SUCCESS [ 42.454 s]
[INFO] Apache James :: Mailbox :: Event :: In VM implementation SUCCESS [02:05 min]
[INFO] Apache James :: Java Mail Extension ................ SUCCESS [  1.015 s]
[INFO] Apache James :: MDN ................................ SUCCESS [  5.351 s]
[INFO] Apache James :: Mailbox :: Store Framework ......... SUCCESS [ 14.953 s]
[INFO] Apache James :: Mailbox :: In Memory ............... SUCCESS [ 11.497 s]
[INFO] Apache James :: Examples ........................... SUCCESS [  0.075 s]
[INFO] custom-listeners ................................... SUCCESS [  6.823 s]
[INFO] Apache James :: Mailetdocs Maven Plugin ............ SUCCESS [ 10.966 s]
[INFO] Apache James :: Basic Mailet Toolkit ............... SUCCESS [  5.079 s]
[INFO] custom-mailets ..................................... SUCCESS [  1.214 s]
[INFO] Apache James :: Mailbox :: Backup .................. SUCCESS [  7.322 s]
[INFO] Apache James :: Server :: Blob ..................... SUCCESS [  0.076 s]
[INFO] Apache James :: Server :: Blob :: API .............. SUCCESS [  3.073 s]
[INFO] Apache James :: Server :: Blob :: Cassandra ........ SUCCESS [03:57 min]
[INFO] Apache James :: Mailbox :: Cassandra ............... SUCCESS [06:30 min]
[INFO] Apache James :: Mailbox :: Tika .................... SUCCESS [ 15.889 s]
[INFO] Apache James :: Mailbox :: ElasticSearch ........... SUCCESS [03:32 min]
[INFO] Apache James :: Mailbox :: Event :: JSON ........... SUCCESS [ 20.987 s]
[INFO] Apache James :: Mailbox :: Event :: In Cassandra implementation SUCCESS [ 43.341 s]
[INFO] Apache James :: Mailbox :: Event :: RabbitMQ implementation SUCCESS [19:36 min]
[INFO] Apache James :: Mailbox :: JPA ..................... SUCCESS [ 20.590 s]
[INFO] Apache James :: Mailbox :: Lucene Index ............ SUCCESS [ 20.569 s]
[INFO] Apache James :: Mailbox :: Maildir ................. SUCCESS [  8.527 s]
[INFO] Apache James :: Server :: Blob :: Memory ........... SUCCESS [ 13.488 s]
[INFO] Apache James :: Server :: Filesystem API ........... SUCCESS [  2.194 s]
[INFO] Apache James :: Server :: Core ..................... SUCCESS [ 10.879 s]
[INFO] Apache James :: Mailbox :: Plugin :: Deleted Messages Vault SUCCESS [  9.607 s]
[INFO] Apache James :: Mailbox :: Plugin :: Deleted Messages Vault :: Cassandra SUCCESS [ 37.887 s]
[INFO] Apache James :: Server :: MailRepository :: API .... SUCCESS [  3.686 s]
[INFO] Apache James :: Server :: Data  :: API ............. SUCCESS [  4.993 s]
[INFO] Apache James :: Server :: DNS Service :: API ....... SUCCESS [  1.011 s]
[INFO] Apache James :: Server :: DNS Service :: Test ...... SUCCESS [  1.130 s]
[INFO] Apache James :: Server :: Data :: Library .......... SUCCESS [  4.904 s]
[INFO] Apache James :: Server :: MailRepository :: Memory . SUCCESS [  2.145 s]
[INFO] Apache James :: Server :: Data  :: MEMORY .......... SUCCESS [  6.195 s]
[INFO] Apache James :: Mailbox :: Plugin :: Quota Mailing . SUCCESS [  5.238 s]
[INFO] Apache James :: Mailbox :: Plugin :: Quota Mailing :: Cassandra SUCCESS [ 33.609 s]
[INFO] Apache James :: Mailbox :: Plugin :: Quota Mailing :: Memory SUCCESS [  4.096 s]
[INFO] Apache James :: Mailbox :: Plugin :: Quota Search :: API SUCCESS [  4.047 s]
[INFO] Apache James :: Mailbox :: Plugin :: Quota Search :: ElasticSearch SUCCESS [ 31.450 s]
[INFO] Apache James :: Mailbox :: Plugin :: Quota Search :: Scanning SUCCESS [  4.315 s]
[INFO] Apache JAMES third party ........................... SUCCESS [  0.102 s]
[INFO] Apache James :: Third Party :: SpamAssassin ........ SUCCESS [ 39.330 s]
[INFO] Apache James :: Mailbox :: Plugin :: SpamAssassin .. SUCCESS [  4.097 s]
[INFO] Apache James :: Mailbox :: Scanning ................ SUCCESS [  7.274 s]
[INFO] Apache James :: Mailbox :: Tools :: Copier ......... SUCCESS [  7.206 s]
[INFO] Apache James :: Mailbox :: Tools :: Indexer ........ SUCCESS [01:26 min]
[INFO] Apache James :: Mailbox :: Spring .................. SUCCESS [  6.022 s]
[INFO] Apache James :: Mailbox :: Tools :: JPA migrator ... SUCCESS [  2.298 s]
[INFO] Apache James :: Mailbox :: Tools :: Maildir utils .. SUCCESS [  1.300 s]
[INFO] Apache James :: AI Mailets ......................... SUCCESS [  1.810 s]
[INFO] Apache James :: Crypto Mailets ..................... SUCCESS [  8.758 s]
[INFO] Apache James :: ICalendar Mailets .................. SUCCESS [  4.496 s]
[INFO] Apache James :: Standard Mailets ................... SUCCESS [ 12.240 s]
[INFO] Apache James :: Metrics :: Dropwizard .............. SUCCESS [  3.383 s]
[INFO] Apache James :: Metrics :: Logger .................. SUCCESS [  3.129 s]
[INFO] Apache James MPT ................................... SUCCESS [  0.064 s]
[INFO] Apache James MPT All ............................... SUCCESS [  0.908 s]
[INFO] Apache James MPT Core .............................. SUCCESS [  8.644 s]
[INFO] Apache James MPT AntLib ............................ SUCCESS [ 26.976 s]
[INFO] Apache James MPT Application ....................... SUCCESS [  5.785 s]
[INFO] Apache James :: Server :: Mailbox Adapter .......... SUCCESS [  3.049 s]
[INFO] Apache James :: Protocols .......................... SUCCESS [  0.085 s]
[INFO] Apache James :: Protocols :: API ................... SUCCESS [  3.131 s]
[INFO] Apache James :: Protocols :: IMAP .................. SUCCESS [ 13.867 s]
[INFO] Apache James MPT Imap Mailbox ...................... SUCCESS [  0.069 s]
[INFO] Apache James MPT Imap Mailbox Core ................. SUCCESS [  1.663 s]
[INFO] Apache James MPT Imap Mailbox - Cassandra .......... SUCCESS [06:58 min]
[INFO] Apache James MPT Imap Mailbox - Cyrus .............. SUCCESS [  1.668 s]
[INFO] Apache James MPT Imap Mailbox - ElasticSearch ...... SUCCESS [ 27.854 s]
[INFO] Apache James MPT Imap Mailbox - External James ..... SUCCESS [  1.367 s]
[INFO] Apache James MPT Imap Mailbox - InMemory ........... SUCCESS [ 12.729 s]
[INFO] Apache James MPT Imap Mailbox - JPA ................ SUCCESS [ 42.395 s]
[INFO] Apache James MPT Imap Mailbox - LuceneSearch ....... SUCCESS [ 16.790 s]
[INFO] Apache James MPT Imap Mailbox - Maildir ............ SUCCESS [ 12.936 s]
[INFO] Apache James MPT Imap Mailbox - RabbitMQ EventBus .. SUCCESS [01:16 min]
[INFO] Apache James :: Protocols :: ManageSieve ........... SUCCESS [  2.550 s]
[INFO] Apache James MPT ManageSieve ....................... SUCCESS [  0.066 s]
[INFO] Apache James MPT ManageSieve Core .................. SUCCESS [  1.264 s]
[INFO] Apache James :: Server :: Data :: Cassandra Persistence SUCCESS [03:16 min]
[INFO] Apache James MPT ManageSieve Cassandra ............. SUCCESS [ 40.405 s]
[INFO] Apache James :: Server :: Data :: File Persistence . SUCCESS [ 30.642 s]
[INFO] Apache James MPT ManageSieve File .................. SUCCESS [  3.084 s]
[INFO] Apache James :: Server :: DNS Service :: Implementation SUCCESS [  3.628 s]
[INFO] Apache James :: Server :: Guice .................... SUCCESS [  0.065 s]
[INFO] Apache James :: Server :: Guice :: Configuration ... SUCCESS [  3.041 s]
[INFO] james-server-guice-utils ........................... SUCCESS [  3.332 s]
[INFO] Apache James :: Server :: Guice :: Mailbox ......... SUCCESS [  4.707 s]
[INFO] Apache James :: Protocols :: Netty Implementation .. SUCCESS [  3.028 s]
[INFO] Apache James :: Server :: Protocols Library ........ SUCCESS [  1.378 s]
[INFO] Apache James :: Server :: IMAP ..................... SUCCESS [  3.678 s]
[INFO] Apache James :: Server :: Guice :: Imap ............ SUCCESS [  1.349 s]
[INFO] Apache James :: Server :: DNS Service :: Library ... SUCCESS [  2.839 s]
[INFO] Apache James :: Server :: Mail Queue :: API ........ SUCCESS [  3.093 s]
[INFO] Apache James :: Server :: Mail Queue :: Memory ..... SUCCESS [ 15.904 s]
[INFO] Apache James :: Protocols :: SMTP .................. SUCCESS [ 13.177 s]
[INFO] Apache James :: Server :: SMTP ..................... SUCCESS [ 18.836 s]
[INFO] Apache James :: Protocols :: LMTP .................. SUCCESS [  8.465 s]
[INFO] Apache James :: Server :: LMTP ..................... SUCCESS [  1.456 s]
[INFO] Apache James :: Server :: Guice :: Lmtp ............ SUCCESS [  1.374 s]
[INFO] Apache James :: Server :: Mailet :: DKIM ........... SUCCESS [  3.760 s]
[INFO] Apache James :: Server :: Mailetcontainer API ...... SUCCESS [  1.183 s]
[INFO] Apache James :: Server :: Mailetcontainer Camel .... SUCCESS [  8.009 s]
[INFO] Apache James :: Server :: Guice :: Mailet .......... SUCCESS [  8.588 s]
[INFO] Apache James :: Server :: Guice :: Netty ........... SUCCESS [  1.013 s]
[INFO] Apache James :: Protocols :: POP3 .................. SUCCESS [  6.270 s]
[INFO] Apache James :: Server :: POP3 ..................... SUCCESS [  9.570 s]
[INFO] Apache James :: Server :: Guice :: Pop ............. SUCCESS [  1.501 s]
[INFO] Apache James :: Server :: Guice :: Smtp ............ SUCCESS [  1.285 s]
[INFO] Apache James :: Server :: Onami .................... SUCCESS [  2.623 s]
[INFO] Apache James :: Server :: Guice Common ............. SUCCESS [  4.406 s]
[INFO] Apache James MPT SMTP .............................. SUCCESS [  0.069 s]
[INFO] Apache James MPT SMTP Core ......................... SUCCESS [  2.014 s]
[INFO] Apache James :: Server :: JWT ...................... SUCCESS [  3.209 s]
[INFO] Apache James :: Server :: Web Admin :: Core ........ SUCCESS [ 14.163 s]
[INFO] Apache James :: Server :: Blob :: Export :: API .... SUCCESS [  2.766 s]
[INFO] Apache James :: Server :: Web Admin :: Mailbox :: Deleted Message Vault SUCCESS [ 20.473 s]
[INFO] Apache James :: Server :: Guice :: Mailbox :: Plugin :: Deleted messages vault SUCCESS [  2.890 s]
[INFO] Apache James :: Server :: Blob Store API - guice injection SUCCESS [  1.038 s]
[INFO] Apache James :: Third Party :: LinShare ............ SUCCESS [01:52 min]
[INFO] Apache James :: Server :: Blob :: Export :: File implementation SUCCESS [  4.284 s]
[INFO] Apache James :: Server :: Blob Exporting Mechanisms - guice injection SUCCESS [  3.998 s]
[INFO] Apache James :: Server :: Data :: JMAP ............. SUCCESS [  8.599 s]
[INFO] Apache James :: Server :: Data :: JMAP :: Cassandra persistance SUCCESS [ 58.475 s]
[INFO] Apache James :: Server :: Mail Queue :: JMS ........ SUCCESS [01:02 min]
[INFO] Apache James :: Server :: Mail Queue :: ActiveMQ ... SUCCESS [01:54 min]
[INFO] Apache James :: Server :: Guice :: ActiveMQ ........ SUCCESS [  1.576 s]
[INFO] Apache James :: Server :: Metrics ElasticSearch Reporter SUCCESS [ 43.533 s]
[INFO] Apache James :: Server :: Guice :: ES metric reporter SUCCESS [  1.218 s]
[INFO] james-server-jmap .................................. SUCCESS [  4.706 s]
[INFO] Apache James :: Server :: JMAP (Draft) ............. SUCCESS [ 23.451 s]
[INFO] Apache James :: Server :: Guice :: JMAP (draft) .... SUCCESS [  4.569 s]
[INFO] Apache James :: Server :: Mailbox :: JMX ........... SUCCESS [  4.622 s]
[INFO] Apache James :: Server :: Guice :: JMX ............. SUCCESS [  3.804 s]
[INFO] Apache James :: Server :: Guice :: Mailbox :: Plugin :: SpamAssassin SUCCESS [  2.614 s]
[INFO] Apache James :: Server :: ManageSieve .............. SUCCESS [  1.319 s]
[INFO] Apache James :: Server :: Guice :: Managedsieve .... SUCCESS [  1.332 s]
[INFO] Apache James :: Server :: Blob :: Common ........... SUCCESS [  1.045 s]
[INFO] Apache James :: Server :: Blob :: Mail store ....... SUCCESS [  3.916 s]
[INFO] Apache James :: Server :: Mail Queue :: RabbitMQ ... SUCCESS [12:05 min]
[INFO] Apache James :: Server :: Guice :: RabbitMQ ........ SUCCESS [  1.429 s]
[INFO] Apache James :: Server :: Guice :: Webadmin ........ SUCCESS [  1.448 s]
[INFO] Apache James :: Server :: Web Admin :: data ........ SUCCESS [ 46.376 s]
[INFO] Apache James :: Server :: Guice :: Webadmin :: Data  SUCCESS [  1.495 s]
[INFO] Apache James :: Server :: Web Admin :: Cassandra ... SUCCESS [  7.978 s]
[INFO] Apache James :: Server :: Web Admin :: Cassandra :: Data SUCCESS [ 35.991 s]
[INFO] Apache James :: Server :: Web Admin :: mailbox ..... SUCCESS [01:20 min]
[INFO] Apache James :: Server :: Web Admin :: JMAP ........ SUCCESS [ 10.848 s]
[INFO] Apache James :: Server :: Guice :: Webadmin :: JMAP  SUCCESS [  1.814 s]
[INFO] Apache James :: Server :: Guice :: Webadmin :: Mailbox SUCCESS [  1.541 s]
[INFO] Apache James :: Server :: Web Admin :: MailQueue ... SUCCESS [ 17.881 s]
[INFO] Apache James :: Server :: Guice :: Webadmin :: MailQueue SUCCESS [  1.266 s]
[INFO] Apache James :: Server :: Web Admin :: MailRepository SUCCESS [ 20.440 s]
[INFO] Apache James :: Server :: Guice :: Webadmin :: MailRepository SUCCESS [  1.335 s]
[INFO] Apache James :: Server :: Web Admin :: Swagger ..... SUCCESS [  7.546 s]
[INFO] Apache James :: Server :: Guice :: Webadmin :: Swagger SUCCESS [  1.263 s]
[INFO] Apache James :: Server :: Mailets .................. SUCCESS [ 20.987 s]
[INFO] Apache James :: Server :: JMAP (draft) :: Integration testing SUCCESS [  0.062 s]
[INFO] Apache James :: Server :: JMAP (draft) :: Integration testing Common SUCCESS [ 12.136 s]
[INFO] Apache James :: Server :: MailRepository :: Cassandra SUCCESS [02:05 min]
[INFO] Apache James :: Server :: Cassandra - guice injection SUCCESS [04:36 min]
[INFO] Apache James MPT SMTP Cassandra .................... SUCCESS [ 56.901 s]
[INFO] Apache James :: Server :: Blob :: Object storage ... SUCCESS [12:24 min]
[INFO] Apache James :: Server :: Blob Store in object storage - guice injection SUCCESS [ 38.474 s]
[INFO] Apache James :: Server :: Blob :: Union Blob Storage SUCCESS [  7.460 s]
[INFO] Apache James :: Server :: Task :: Distributed ...... SUCCESS [08:16 min]
[INFO] Apache James :: Server :: Cassandra with RabbitMQ - guice injection SUCCESS [07:17 min]
[INFO] Apache James MPT SMTP Cassandra RabbitMQ Object Storage SUCCESS [01:17 min]
[INFO] Apache James MPT Maven2 Plugin ..................... SUCCESS [  6.985 s]
[INFO] Apache James :: Server :: Client ................... SUCCESS [  4.293 s]
[INFO] Apache James :: Server :: Data :: JDBC Persistence . SUCCESS [ 17.965 s]
[INFO] Apache James :: Server :: Data :: JPA Persistence .. SUCCESS [ 10.195 s]
[INFO] Apache James :: Server :: Data :: LDAP Implementation SUCCESS [ 22.876 s]
[INFO] Apache James :: Server :: FetchMail ................ SUCCESS [  2.535 s]
[INFO] Apache James :: Server :: Mail Queue :: File ....... SUCCESS [  2.868 s]
[INFO] Apache James :: Server :: Container Spring ......... SUCCESS [  4.929 s]
[INFO] Apache James :: Server :: App ...................... SUCCESS [ 35.319 s]
[INFO] Apache James :: Server :: Blob Memory - guice injection SUCCESS [  1.078 s]
[INFO] Apache James :: Server :: Guice :: Sieve :: File ... SUCCESS [  1.299 s]
[INFO] Apache James :: Server :: Memory - guice injection . SUCCESS [01:38 min]
[INFO] Apache James :: Server :: Cli :: Integration ....... SUCCESS [ 26.181 s]
[INFO] Apache James :: Server :: Cassandra/Ldap - guice injection SUCCESS [01:24 min]
[INFO] Apache James :: Server :: Cassandra/Ldap with RabbitMQ - guice injection SUCCESS [02:51 min]
[INFO] Apache James :: Server :: JPA - guice common ....... SUCCESS [  4.697 s]
[INFO] Apache James :: Server :: Guice :: Sieve :: JPA .... SUCCESS [  1.436 s]
[INFO] Apache James :: Server :: JPA - guice injection .... SUCCESS [ 29.257 s]
[INFO] Apache James :: Server :: JPA + SMTP - common guice injection SUCCESS [  2.425 s]
[INFO] Apache James :: Server :: JPA + SMTP - guice injection SUCCESS [ 10.570 s]
[INFO] Apache James :: Server :: JPA + SMTP + mariaDB client - guice injection SUCCESS [ 24.287 s]
[INFO] Apache James :: Server :: Guice :: testing ......... SUCCESS [  0.065 s]
[INFO] Apache James :: Server :: Guice :: Mailet :: Custom testing mailets :: dependency SUCCESS [  1.012 s]
[INFO] Apache James :: Server :: Guice :: Mailet :: Custom testing mailets SUCCESS [  5.928 s]
[INFO] custom-mailets-implementation ...................... SUCCESS [  1.139 s]
[INFO] Apache James :: Server :: Mailet :: Mock SMTP Server SUCCESS [ 58.657 s]
[INFO] Apache James :: Server :: Mailets Integration Testing SUCCESS [03:02 min]
[INFO] Apache James :: Server :: JMAP RFC 8621 ............ SUCCESS [ 52.140 s]
[INFO] Apache James :: Server :: JMAP (draft) :: Cassandra Integration testing SUCCESS [04:57 min]
[INFO] Apache James :: Server :: JMAP (draft) :: Memory Integration testing SUCCESS [07:31 min]
[INFO] Apache James :: Server :: JMAP (draft) :: RabbitMQ + Object Store + Cassandra Integration testing SUCCESS [07:56 min]
[INFO] Apache James :: Server :: Web Admin ................ SUCCESS [  0.071 s]
[INFO] Apache James :: Server :: Web Admin server integration tests SUCCESS [  0.067 s]
[INFO] Apache James :: Server :: Web Admin server integration tests :: Common SUCCESS [  3.402 s]
[INFO] Apache James :: Server :: Web Admin server integration tests :: Distributed SUCCESS [11:51 min]
[INFO] Apache James :: Server :: Web Admin server integration tests :: Memory SUCCESS [03:16 min]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  03:09 h
[INFO] Finished at: 2020-04-01T10:02:03Z
[INFO] ------------------------------------------------------------------------
```

chibenwa commented:
To be improved (IMO) 

```
[INFO] Apache James :: Server :: Mail Queue :: RabbitMQ ... SUCCESS [12:05 min]
[INFO] Apache James :: Mailbox :: Event :: RabbitMQ implementation SUCCESS [19:36 min]
```

trantienduchn commented:
I remember the old builds took 2:50 mins intotal total

chibenwa commented:
> I remember the old builds took 2 hours 50 mins in total

Does it mean we can gain 20 minutes?

trantienduchn commented:
> Does it mean we can gain 20 minutes?

Sounds like yeah, after fasten some rabbitMQ related modules. And this little one (In VM) it tooks 2 minutes while the cassandra version took only 43 seconds.

```
[INFO] Apache James :: Mailbox :: Event :: In VM implementation SUCCESS [02:05 min]
[INFO] Apache James :: Mailbox :: Event :: In Cassandra implementation SUCCESS [ 43.341 s]
[INFO] Apache James :: Mailbox :: Event :: RabbitMQ implementation SUCCESS [19:36 min]
```

chibenwa commented:
Merged
